tests/virt_autotest/virt_utils.pm

```diff
@@ -81,20 +81,36 @@ sub locate_sourcefile {
 }
 
 sub get_repo_0_prefix {
-    # Get customized repo location from REPO_0_PREFIX and append missing forward slash to the end
+    my ($default_repo_urls_ref) = @_;
+    my @default_repo_urls = @{$default_repo_urls_ref};
     my $repo_0_prefix = get_var("REPO_0_PREFIX", "");
     $repo_0_prefix .= '/' unless $repo_0_prefix =~ /\/$/;
-    $repo_0_prefix = ($repo_0_prefix ne "/" ? $repo_0_prefix : "http://openqa.suse.de/assets/repo/");
-    return $repo_0_prefix;
+    foreach my $url (@default_repo_urls) {
+        my $repo_0 = get_var("REPO_0");
+        record_info("log1", $repo_0);
+        record_info("log2", get_version_for_daily_build_guest);
+        $repo_0 =~ s/-Online-/-Full-/ if (get_version_for_daily_build_guest =~ /15-sp[2-9]/i);
+        record_info("log3", $repo_0);
+        my $repo_0_prefix = ($repo_0_prefix ne "/" ? $repo_0_prefix : "http://$url/assets/repo/");
+        # Check the HTTP response for this URL
+        my $result = script_run("wget --spider -q '$repo_0_prefix$repo_0/CHECKSUMS'");
```

Comment:
I think ```curl -sSf url``` is better, because it has more widespread availablity.

Reply:
Done.